api/resolvers/item.js

```diff
@@ -803,15 +803,46 @@ export default {
 
       const [{ item_act: vote }] = await serialize(models, ...calls)
 
-      const updatedItem = await models.item.findUnique({ where: { id: Number(id) } })
-      const title = `your ${updatedItem.title ? 'post' : 'reply'} ${updatedItem.fwdUser ? 'forwarded' : 'stacked'} ${
-        numWithUnits(msatsToSats(updatedItem.msats))}${updatedItem.fwdUser ? ` to @${updatedItem.fwdUser.name}` : ''}`
-      sendUserNotification(updatedItem.userId, {
-        title,
-        body: updatedItem.title ? updatedItem.title : updatedItem.text,
-        item: updatedItem,
-        tag: `TIP-${updatedItem.id}`
-      }).catch(console.error)
+      const notify = async () => {
+        try {
+          const updatedItem = await models.item.findUnique({ where: { id: Number(id) } })
+          const forwards = await models.itemForward.findMany({ where: { itemId: Number(id) } })
+          const userPromises = forwards.map(fwd => models.user.findUnique({ where: { id: fwd.userId } }))
+          const userResults = await Promise.allSettled(userPromises)
+          const mappedForwards = forwards.map((fwd, index) => ({ ...fwd, user: userResults[index].value ?? null }))
+          let forwardedSats = 0
+          let forwardedUsers = ''
+          if (mappedForwards.length) {
+            forwardedSats = Math.floor(msatsToSats(updatedItem.msats) * mappedForwards.map(fwd => fwd.pct).reduce((sum, cur) => sum + cur) / 100)
+            forwardedUsers = mappedForwards.map(fwd => `@${fwd.user.name}`).join(', ')
```

Comment:
What about just showing the number of forwards in the push notification title?

Instead of 

`your post forwarded 1000 sats to @k00b, @WeAreAllSatoshi, @ekzyis`

just

`your post forwarded 1000 sats to 3 stackers`

Reply:
> What about just showing the number of forwards in the push notification title?
> 
> Instead of
> 
> `your post forwarded 1000 sats to @k00b, @WeAreAllSatoshi, @ekzyis`
> 
> just
> 
> `your post forwarded 1000 sats to 3 stackers`
> 
> edit: except when it's a single forward. then the name can be used

Just curious, what is the concern here? notification length? complexity?

Reply:
> Also, I guess this was missing the whole time: there are no notifications sent to the forwarded stackers

Yea, that would be a nice enhancement

Reply:
Notification length + consistency with other notifications but it's not that important. We can keep it as is for now. Just wanted to mention